scripts/create_test_data.R

```diff
@@ -4,7 +4,7 @@
 ## We read some input data and prepare a set of demographic projection
 ## and HIV parameters for both the adult and the child model.
 ## We also run leapfrog and save out the result for use in reference tests
-
+rm(list = ls())
```

Comment:
Is this some left over temporary change? Or you want this for running interactively?